src/content/en/showcase/2016/iowa2016.md

```diff
@@ -389,15 +389,15 @@ When users navigate to a new page, a sequence of things happen:
 
 1. The top nav slides a selection bar to the new link.
 - The page's heading fades away.
-- The page content slides down then fades away.
-- Using the reverse animation, the new page's heading and content appear.
+- The page's content slides down then fades away.
+- Using reverse animations, the new page's heading and content appear.
```

Comment:
This doesn't sound right to me. I want to emphasize the animation is the same, but it's reversed.